standby/src/lib.rs

```diff
@@ -55,8 +55,8 @@
 //!
 //! let standby = Standby::new();
 //!
-//! let message = standby.wait_for_message(ChannelId(123), |event: &MessageCreate| {
-//!     event.author.id == UserId(456) && event.content == "test"
+//! let message = standby.wait_for_message(ChannelId::new(123).expect("non zero"), |event: &MessageCreate| {
+//!     event.author.id == UserId::new(456).expect("non zero") && event.content == "test"
```

Comment:
I suppose this could just use `event.author.id.get() == 456` to be more concise.